### PR TITLE
GUNDI-3065: EULA

### PR DESCRIPTION
**Pull request by marianobrc**
### What does this PR do?
- Adds models to store and accept EULAs
- Adds endpoints to retrieve and accept EULA
- Adds basic test coverage for EULA endpoints
- Adds classes to manage EULAs through the Django admin UI

### Relevant link(s)
[GUNDI-3065](https://allenai.atlassian.net/browse/GUNDI-3065)
[Docs](https://allenai.atlassian.net/wiki/spaces/CDIP/pages/29635248135/Gundi+API+v2#EULA)

[GUNDI-3065]: https://allenai.atlassian.net/browse/GUNDI-3065?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cdip_admin/api/v2/serializers.py [hunk 1]
```diff
@@ -1010,3 +1020,53 @@ class ActivityLogRetrieveSerializer(serializers.Serializer):
...
+        fields = (
+            "user",
+            "eula",
+            "accept",
+            "date_accepted",
+        )
+
+    def get_unique_together_validators(self):
+        # Overriden to disable unique together check as it's handled in the create method
+        return []
```
Comment (davidovichmarcos) [inline cdip_admin/api/v2/serializers.py:1042]: nitpick: do we have a return type like not implemented? (not sure if it's an option)

**Review by davidovichmarcos — APPROVED** [review]
LGTM!

**Review by vgarcia13 — APPROVED** [review]

File: cdip_admin/api/v2/serializers.py (hunk 1 above)
Comment (marianobrc) [reply cdip_admin/api/v2/serializers.py:1042]: Thanks for the suggestion. If the base class was part of our code, we could consider raising `NotImplementedError` or handling that case in other ways. But here we are overriding a method implemented in a base class that is part of the `django-rest-framework` so we can't / don't want to change how that works. Here I wanted to disable some validators, and the way of doing that with DRF is to return that empty list.

**Review by chrisdoehring — APPROVED** [review]
This looks great.

**Pull request merged**